changeset.go

```diff
@@ -87,7 +88,7 @@ func (c *Changeset) Apply(doc *rel.Document, mut *rel.Mutation) {
 				c.applyAssocMany(doc, field, mut, v)
 			}
 		default:
-			if field != pField && scannable(c.types[field]) {
+			if (pField != field || pField == field && !c.ignorePrimary) && scannable(c.types[field]) { //if not PK - try to set
```

Comment:
maybe comment can be removed or updated?

Reply:
Comment is removed